### DISCUSSION TITLE
Not Syncing, Plots not available

### DISCUSSION THREAD

**Discussion by papsphilip** [Chia-Network/chia-blockchain#6383] [Support]
I hadnt mined with chia for a few months, now i opened the program and its not syncing also when i look at my plots it says that they are all unavailable. Although i havent deleted anything, the plots are still on my HDD. Is this a bug or somekind of update i need to do?

**Comment by joelrb** [comment]
> I hadnt mined with chia for a few months, now i opened the program and its not syncing also when i look at my plots it says that they are all unavailable. Although i havent deleted anything, the plots are still on my HDD. Is this a bug or somekind of update i need to do?

What do you mean hadn't mined for a few months? Chia Mainnet (1.0) was officially started (lauched) last March 19. If you were mining before that date that means you were in Testnet. Your plots then won't work in Mainnet. You need to re-plot for Mainnet. If you plotted from March 19 onwards, then your plots should still be valid and maybe you need to update to latest Chia version.

**Comment by BrandtH22** [comment]
@papsphilip , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
